tangle-gradle-plugin/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ import tangle.builds.VERSION_NAME
 plugins {
   javaLibrary
   id("java-gradle-plugin")
-  id("com.gradle.plugin-publish") version "1.0.0"
+  id("com.gradle.plugin-publish") version "1.1.0"
```

Comment:
This wasn't necessary to build but [the changelog](https://plugins.gradle.org/plugin/com.gradle.plugin-publish) mentioned a bug around tags so I upgraded anyway